ElementX/Sources/Screens/Timeline/TimelineViewModel.swift

```diff
@@ -258,6 +258,39 @@ class TimelineViewModel: TimelineViewModelType, TimelineViewModelProtocol {
     
     // MARK: - Private
     
+    private func handleTappedOnSenderDetails(userID: String) {
+        // We also need to make sure the user is in the joined state, otherwise we jusr show the details.
+        guard let memberProxy = roomProxy.membersPublisher.value.first(where: { $0.userID == userID && $0.membership == .join }) else {
+            actionsSubject.send(.displaySenderDetails(userID: userID))
+            return
+        }
+        
+        if memberProxy.role != .administrator, state.canCurrentUserBan || state.canCurrentUserKick {
```

Comment:
Check with Android, but as we're moving back to better alignment with web/the matrix spec, we should probable change this to something like
```suggestion
        if memberProxy.powerLevel < currentUserProxy.powerLevel, state.canCurrentUserBan || state.canCurrentUserKick {
```